src/ui/public/doc_table/doc_table.html

```diff
@@ -104,6 +104,11 @@
   <kbn-infinite-scroll ng-if="infiniteScroll" more="addRows"></kbn-infinite-scroll>
 </div>
 
+<div ng-if="!columns.length" class="table-vis-error">
+  <h2><i class="fa fa-meh-o"></i></h2>
```

Comment:
I remember @cjcenizal telling me to not use `<i>` for the icons, but a `span` with the `kuiIcon` class...